src/uucore/src/lib/features/format/argument.rs

```diff
@@ -41,14 +41,7 @@ impl<'a, T: Iterator<Item = &'a FormatArgument>> ArgumentIter<'a> for T {
         match next {
             FormatArgument::Char(c) => *c,
             FormatArgument::Unparsed(s) => {
-                let mut chars = s.chars();
-                let Some(c) = chars.next() else {
-                    return '\0';
-                };
-                let None = chars.next() else {
-                    return '\0';
-                };
-                c
+                s.chars().next().unwrap_or('\0')
```

Comment:
I would prefer to "fix" it in a second phase, because you might end up breaking up a multi-bytes UTF8 sequence and we have to be careful about this if it ends up in a `str`.

Reply:
I turned this into an issue: https://github.com/uutils/coreutils/issues/5826